documents-webapp/src/main/webapp/skin/less/documents.less

```diff
@@ -133,7 +133,7 @@
     .documents-header-right-actions {
       .inputDocumentsFilter {
         width: auto;
-        margin-top: 6px !important;
+        margin-top: 8px !important;
```

Comment:
You can add vuetify helper class **mt-2** directly

Reply:
good catch thanks